landing-pages/site/content/en/blog/airflow-survey-2023/index.md

```diff
@@ -2,6 +2,10 @@
 title: "Airflow Survey 2023"
 linkTitle: "Airflow Survey 2023"
 tags: ["community", "survey", "users"]
+date: "2023-09-21"
+author: "Ankit Chaurasia"
+github: "sunank200"
+linkedin: "sunank200"
```

Comment:
Should we use the original author of the page as the author here: https://github.com/apache/airflow-site/pull/923 since we back dated it

Reply:
Changed it.